spec/controllers/posts_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :controller do
```

Comment:
- [ ] For these tests, you are actually testing the requests and not the controller actions. THe Rspec type is meant to be request `type: request`. I'll suggest you update this correctly. `RSpec.describe 'Posts', type: :request do`.

- [ ] Also these tests are meant to be in the `requests` folder in your spec file and not in the `controllers` folder. I'll suggest you update these and move these tests to the correct folder location. :smile: :small_airplane: